docs/sources/enterprise/usage-insights/export-logs.md

```diff
@@ -0,0 +1,84 @@
++++
+title = "Export logs of usage insights"
+description = "Export logs of usage insights"
+keywords = ["grafana", "export", "usage-insights", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/export-logs.md"]
+weight = 500
++++
+
+# Export logs of usage insights
+
+> **Note:** Available in Grafana Enterprise v7.4+.
+
+By exporting usage logs to Loki, you can directly query them and create dashboards of the information that matters to you most, such as dashboard errors, most active organizations, or your top-10 most-used queries. Currently, usage logs can only be exported to Loki.
+
+## Usage insights logs
+
+Usage insights logs are JSON objects that represent various user activity:
+- A user opens a dashboard.
+- A query is sent to a data source.
+
+### Scope
+
+A log is created every time a user opens a dashboard or when a query is sent to a data source in the dashboard view. A query that is made through Explore does not generate a log.
```

Comment:
```suggestion
A log is created every time a user opens a dashboard or when a query is sent to a data source in the dashboard view. A query that is performed via Explore does not generate a log.
```

Reply:
My gut is that _perform_ is the more "natural" verb in this context because it is about place of execution rather than the specifics of creating a query.